### ISSUE TITLE
Good job

### ISSUE THREAD

**Issue by abuabdelrahman1413** [BogdanCozma14/LandingPage#1]
but you have some issues need to fix i'll be glad if i can help you

**Comment by BogdanCozma14** [comment]
Hello. I am keen to learn from my mistakes, so I'd happily listen to suggestions.

**Comment by abuabdelrahman1413** [comment]
Yes, keep working hard. This is the mindset of a successful person.
